<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-10-01T11:43:22 IST" name="Default suite" finished-at="2023-10-01T11:43:34 IST" duration-ms="12043">
    <groups>
    </groups>
    <test started-at="2023-10-01T11:43:22 IST" name="Default test" finished-at="2023-10-01T11:43:34 IST" duration-ms="12043">
      <class name="nucotjulypomseries.tests.LoginPageTest">
        <test-method is-config="true" signature="loginSetUp()[pri:0, instance:nucotjulypomseries.tests.LoginPageTest@2474f125]" started-at="2023-10-01T11:43:22 IST" name="loginSetUp" finished-at="2023-10-01T11:43:29 IST" duration-ms="7427" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginSetUp -->
        <test-method signature="doLoginTest()[pri:0, instance:nucotjulypomseries.tests.LoginPageTest@2474f125]" started-at="2023-10-01T11:43:29 IST" name="doLoginTest" finished-at="2023-10-01T11:43:33 IST" duration-ms="4286" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doLoginTest -->
        <test-method signature="getLoginPageCurrentUrlTest()[pri:0, instance:nucotjulypomseries.tests.LoginPageTest@2474f125]" started-at="2023-10-01T11:43:33 IST" name="getLoginPageCurrentUrlTest" finished-at="2023-10-01T11:43:33 IST" duration-ms="19" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at nucotjulypomseries.tests.LoginPageTest.getLoginPageCurrentUrlTest(LoginPageTest.java:18)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getLoginPageCurrentUrlTest -->
        <test-method signature="getLoginPageTitleTest()[pri:0, instance:nucotjulypomseries.tests.LoginPageTest@2474f125]" started-at="2023-10-01T11:43:33 IST" name="getLoginPageTitleTest" finished-at="2023-10-01T11:43:33 IST" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getLoginPageTitleTest -->
        <test-method is-config="true" signature="closeApp()[pri:0, instance:nucotjulypomseries.tests.LoginPageTest@2474f125]" started-at="2023-10-01T11:43:33 IST" name="closeApp" finished-at="2023-10-01T11:43:34 IST" duration-ms="191" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- nucotjulypomseries.tests.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
